lib/Config.ts

```diff
@@ -17,8 +17,7 @@ class Config {
   public logdateformat: string;
   public network: Network;
   public rpc: { disable: boolean, host: string, port: number };
-  public lndbtc: LndClientConfig;
-  public lndltc: LndClientConfig;
+  public lnd: { [currency: string]: LndClientConfig | undefined } = {};
```

Comment:
I had this thought too - I'll rename the cli code to use `currency` as well. I think "ticker" is actually somewhat ambiguous in that it can refer both to a currency symbol as well as a trading pair symbol. 

The advantage of ticker is that it's shorter for purposes of printing out the command line args for `xud` but I don't think it's worth the confusion.